packit/base_git.py

```diff
@@ -120,6 +120,10 @@ def specfile(self) -> Specfile:
             self._specfile = Specfile(
                 self.absolute_specfile_path,
                 sourcedir=self.absolute_source_dir,
+                macros=[
+                    (str(k), v if v is None else str(v))
```

Comment:
aren't these strings already? (Could you please add some testcase having the `parse_time_macros` non-empty?)

Reply:
> That depends on YAML parser, both keys (unlikely but possible) and values could be interpreted as numbers.

got it, thanks for the explanation, maybe worth a comment then

> What exactly should it test? Just that a macro expands to an expected value?

I meant to cover this part of the code, but if you add a comment I am ok with no test